addon/components/file-editor/component.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import layout from './template';
+import ace from 'ember-ace';
+
+/**
+ * @module ember-osf
+ * @submodule components
+ */
+
+/**
+ * Edit the chosen file
+ *
+ * Sample usage:
+ * ```handlebars
+ * {{file-editor
+     fileText=fileText
+     save='save'}}
+ * ```
+ * @class file-editor
+ */
+
+export default Ember.Component.extend({
+    layout,
+    fileText: '',
+
+    actions: {
+        revertText() {
+            const editor = ace.edit(document.querySelector('[data-ember-ace]'));
```

Comment:
Instead of pulling this out of the DOM, should use the [`ready` action](https://github.com/dfreeman/ember-ace#core) to set the editor.